Core/Core/Modules/ModuleList/CustomCellBackgroundView.swift

```diff
@@ -0,0 +1,34 @@
+//
```

Comment:
This file would be at a better place in `Core/UIViews` folder instead of the modules one.